src/Carnac.Logic/Carnac.Logic.csproj

```diff
@@ -16,6 +16,7 @@
     <TargetFrameworkProfile>
     </TargetFrameworkProfile>
     <NuGetPackageImportStamp>417f57c2</NuGetPackageImportStamp>
+    <RestorePackages>true</RestorePackages>
```

Comment:
This is the old way of doing NuGet package restore. Visual studio will do it automatically or you use `nuget install` from the command line